### DISCUSSION TITLE
How to use Bootstrap 4's “site\docs\4.4\examples” Folder contents using Jekyll on GitHub Pages?

### DISCUSSION THREAD

**Discussion by ColstonBod-oy** [community/community#22156] [Other Feature Feedback, Questions, & Ideas]
<p>In the examples folder of <strong>Bootstrap 4</strong> , I was wondering on how to use and view its contents as everytime I open an <strong>index.html</strong> file to my browser it looks like this:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/3/a3b1bef5930938e8767aaf1a819369c1bd0c13c3.png" data-download-href="/uploads/short-url/nm6E5uXcKbT5z2E04hDGW7sB9e3.png?dl=1" title="Capture4" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/75562733/181069259-5cff07a8-ecb5-4b2a-a49e-374a7283876a.png" alt="Capture4" data-base62-sha1="nm6E5uXcKbT5z2E04hDGW7sB9e3" width="690" height="467" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a3b1bef5930938e8767aaf1a819369c1bd0c13c3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture4</span><span class="informations">989×670 36 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I know I have to use <strong>Jekyll</strong> for this but how am I supposed to use it to view the documents properly on <strong>GitHub Pages</strong>? Am I supposed to install <strong>Jekyll</strong> first?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! Check out this article: <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll" class="inline-onebox" rel="noopener nofollow ugc">Testing your GitHub Pages site locally with Jekyll - GitHub Docs</a> for details on testing and running your GitHub Pages site locally with Jekyll.</p>
